src/sources/util/grpc/mod.rs

```diff
@@ -35,7 +35,7 @@ where
     let listener = tls_settings.bind(&address).await?;
     let stream = listener.accept_stream();
 
-    info!(message = "Building gRPC server.", address = %address);
+    debug!(%address, "Building gRPC server.");
```

Comment:
It was intentionally matching the http server, https://github.com/vectordotdev/vector/blob/master/src/sources/util/http/prelude.rs#L153

Reply:
Hmm, fair. I can revert this... although separately, it seems extraneous to have being logged in the first place. 🤷🏻

Reply:
I'm fine either way- we should probably be consistent in both places.
I had thought the info level was intentional to help debugging user issues. But if that's not the case I def yield to yall's discretion.

Reply:
I was prompted to add it because the grpc had not been listening on the port I expected when working on the `otel` source.  The choice of log level was based on existing behavior, all other logs regarding "listening at blah" are at INFO

Reply:
Unwound this change in 69cc1b51f.